src/sprite.cpp

```diff
@@ -311,6 +336,20 @@ static bool strccmp(std::string_view first, std::string_view second) {
 
     if (!patch(sprite_patch, rom))
         return false;
+
+    if (cfg.SymbolsType.length()) {
+        std::string symbols_to_file = std::string(spr->asm_file) + "." + cfg.SymbolsType;
+        std::string symbols_from_file = std::string(TEMP_SPR_FILE) + "." + cfg.SymbolsType;
+        try {
+            fs::rename(symbols_from_file, symbols_to_file);
+        }
+        catch (const fs::filesystem_error& err) {
+            io.error("Trying to rename \"%s\" returned \"%s\", aborting insertion\n", symbols_from_file.c_str(),
+                     err.what());

```

Comment:
Since now we're using `fs::path` above we can't call `c_str()` on it to print, so I suggest something like this
```c++
io.error("Trying to rename symbol file of \"%s\" returned \"%s\", aborting insertion\n", spr->asm_file,
          err.what());
```